src/hotspot/os/posix/os_posix.cpp

```diff
@@ -87,6 +89,14 @@
 #define assert_with_errno(cond, msg)    check_with_errno(assert, cond, msg)
 #define guarantee_with_errno(cond, msg) check_with_errno(guarantee, cond, msg)
 
+static jlong initial_time_count = 0;
+
+#if defined(_ALLBSD_SOURCE)
+static int clock_tics_per_sec = CLK_TCK;
+#else
+static int clock_tics_per_sec = 100;
+#endif
```

Comment:
Can't we just defer the conditional initialization to the init() function where we already have to conditionally initialise.